<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TuntapWidget</class>
 <widget class="QWidget" name="TuntapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QDockWidget" name="dockWidget_tuntap">
     <property name="features">
      <set>QDockWidget::NoDockWidgetFeatures</set>
     </property>
     <property name="windowTitle">
      <string>Tuntap settings</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents_tuntap">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="layout_mode">
         <item>
          <widget class="QLabel" name="label_mode">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_mode">
           <property name="toolTip">
            <string>Either ‘tun’, or ‘tap’</string>
           </property>
           <item>
            <property name="text">
             <string>tun</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tap</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_user">
         <item>
          <widget class="QLabel" name="label_user">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>User</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_user">
           <property name="toolTip">
            <string>The owning user of the tun/tap interface</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_group">
         <item>
          <widget class="QLabel" name="label_group">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_group">
           <property name="toolTip">
            <string>The owning group of the tun/tap interface</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
